packages/typeorm/src/typeorm.manager.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  DataSource,
+  DataSourceOptions,
+  EntityManager,
+  EntityMetadata,
+  Repository,
+  TreeRepository,
+  MongoRepository,
+} from 'typeorm'
+import { Container } from 'inversify'
+import { createDataSource, getDataSourceToken, getEntityManagerToken, getRepositoryToken } from './typeorm.utils'
+import { TypeOrmOptions } from './interfaces/typeorm-options.interface'
+import { EntityClassOrSchema } from './interfaces/entity-class-or-schema.type'
+import { DEFAULT_DATA_SOURCE_NAME } from './typeorm.constants'
+
+export const container = new Container()
+
+export type DataSourceProvider = () => Promise<DataSource>
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type TypeOrmRepository = Repository<any> | TreeRepository<any> | MongoRepository<any>
+
+export class TypeOrmManager {
+  /**
+   * Bind DataSource and EntityManager to the container.
+   * @param options
+   */
+  static async importRoot(options: TypeOrmOptions): Promise<void> {
+    const dataSource = await createDataSource(options)
+
+    const dataSouceToken = getDataSourceToken(options as DataSourceOptions)
+    container.bind<DataSource>(dataSouceToken).toConstantValue(dataSource)
+
+    const entityManagerToken = getEntityManagerToken(options as DataSourceOptions)
+    container.bind<EntityManager>(entityManagerToken).toConstantValue(dataSource.manager)
+  }
+
+  /**
+   * Binds the repository for each entity in `entities` to the container.
+   * @param entities
+   * @param dataSource
+   */
+  static async importRepository(
+    entities: EntityClassOrSchema[] = [],
+    dataSource: DataSource | DataSourceOptions | string = DEFAULT_DATA_SOURCE_NAME,
+  ): Promise<void> {
+    const conn = this.getDataSource(dataSource)
+
+    ;(entities || []).forEach((entity: EntityClassOrSchema) => {
```

Comment:
```suggestion
    entities?.forEach((entity: EntityClassOrSchema) => {
```

Reply:
Error will be reported if there is no semicolon.

Reply:
It seems no error with `entities?`

Reply:
After testing, `entities?` is right. And semicolon is unnecessary.